package.json

```diff
@@ -31,6 +30,10 @@
     {
       "name": "Chunpeng Huo",
       "url": "https://github.com/3cp"
+    },
+    {
+      "name": "Fisker Cheung",
+      "url": "https://github.com/fisker"
```

Comment:
I didn't contribute much to the source, I'd prefer not list myself here. I don't want see this field because huge in future.

Reply:
OK :-)